aw_client/client.py

```diff
@@ -84,6 +84,11 @@ def replace_last_event(self, bucket, event: Event):
         data = event.to_json_dict()
         self.dispatch_thread.add_request(endpoint, data)
 
+    def heartbeat(self, bucket, event: Event, pulsetime: float):
+        endpoint = "buckets/{}/heartbeat?pulsetime={}".format(bucket, pulsetime)
+        data = event.to_json_dict()
+        self.dispatch_thread.add_request(endpoint, data)
```

Comment:
The only issue with this is that if aw-server is unavailable it will stack a lot of heatbeats, however we could technically just stack a few events with longer pulsetimes instead once the server is back online.

This is fine though since this issue is also present in the normal non-heartbeat event dispatching, I just wanted to mention that there is room for optimization.

I approve